Connector/server.py

```diff
@@ -1,158 +1,81 @@
 #!/usr/bin/python3
-import aiohttp
-import aiohttp_cors
 from aiohttp import web
+import aiohttp_cors
 import importlib
-import json
-import os
-import utils
+from httputils import middleware
+from httputils.router import Router
+from httputils.app import App, appModules
+from httputils.constants import JSON_CONTENT_TYPE
+from rpcutils import middleware as rpcMiddleware
+from wsutils import broker
 from logger import logger
-from rpcutils import rpcutils, errorhandler as rpcErrorHandler
-from webapp import WebApp
-from wsutils import wsutils
-from wsutils.subscribers import WSSubscriber
-from wsutils.broker import Broker
-
-logger.printInfo(f"Loading connector for {os.environ['COIN']}")
-importlib.__import__(os.environ['COIN'].lower())
-
-
-async def rpcServerHandler(request):
-
-    reqParsed = None
-
-    try:
-
-        reqParsed = rpcutils.parseRpcRequest(await request.read())
-        logger.printInfo(f"New RPC request received: {reqParsed}")
-
-        if reqParsed[rpcutils.METHOD] in rpcutils.RPCMethods:
-            payload = rpcutils.RPCMethods[reqParsed[rpcutils.METHOD]](reqParsed[rpcutils.ID], reqParsed[rpcutils.PARAMS])
-        else:
-            logger.printError(f"RPC Method not supported for {os.environ['COIN']}")
-            raise rpcErrorHandler.MethodNotAllowedError(f"RPC Method not supported for {os.environ['COIN']}")
-
-        response = rpcutils.generateRPCResultResponse(
-            reqParsed[rpcutils.ID],
-            payload
-        )
-
-        logger.printInfo(f"Sending response to requestor: {response}")
-
-        return web.Response(
-            text=json.dumps(
-                response
-            ),
-            content_type=rpcutils.JSON_CONTENT_TYPE,
-        )
-
-    except rpcErrorHandler.Error as e:
-
-        response = rpcutils.generateRPCErrorResponse(
-            reqParsed[rpcutils.ID] if reqParsed is not None else rpcutils.UNKNOWN_RPC_REQUEST_ID,
-            e.jsonEncode()
-        )
+# from utils import utils
```

Comment:
Commented line should be deleted

Reply:
The package utils is going to be mandatory once the script is refactored